concord-server/server/src/test/java/com/rabidgremlin/concord/functions/GetEligiblePhrasesForCompletionTest.java

```diff
@@ -51,16 +47,16 @@ public void setUp()
 
     phraseVotes.add(new GroupedPhraseVote("126", "CelineDion", "And my heart will go on and on", 1));
 
-    functionUnderTest = new GetEligiblePhrasesForCompletion(votesDaoMock, phraseVotes, 2);
+    phraseVotes.add(new GroupedPhraseVote("127", "TaylorSwift", "Last Christmas I gave your my heart", 3));
+
+    phraseVotes.add(new GroupedPhraseVote("127", "JustinBieber", "Last Christmas I gave your my heart", 2));
```

Comment:
I don't remember Justin singing this one?

Reply:
No, he didn't, which is why those votes are wrong. It didn't have the case where there is not enough vote differences